general/administration/migrate.md

```diff
@@ -1,13 +1,11 @@
 ---
-uid: admin-migrate-from-emby
+uid: admin-migrate
 title: Migrating
 ---
 
-# Migrating from Emby to Jellyfin
+# Migrating
 
-Direct database migration from Emby (of any version) to Jellyfin is NOT SUPPORTED. We have found many subtle bugs due to the inconsistent database schemas that result from trying to do this, and strongly recommend that all Jellyfin users migrating from Emby start with a fresh database and library scan.
-
-The original procedure is provided below for reference however we cannot support it nor guarantee that a system upgraded in this way will work properly, if at all. If anyone is interested in writing a database migration script which will correct the deficiencies in the existing database and properly import them into Jellyfin, [we would welcome it however](xref:contrib-index)!
+It is possible to migrate your system to another system by using Environmentals. It's possible to do this via the command line or using Docker Environmentals. To read more, see the [Configuration](https://jellyfin.org/docs/general/administration/configuration.html) page.
```

Comment:
```suggestion
It is possible to migrate your system to another system by using environment variables. It's possible to do this via the command line or by using Docker environment variables. To read more, see the [Configuration](https://jellyfin.org/docs/general/administration/configuration.html) page.
```

Not sure if the intentional meaning is `environment variables` or `environments`